src/app-layer-ftp.c

```diff
@@ -424,29 +473,29 @@ static int FTPGetLine(FtpState *state)
  * transferred to the ftp server
  * \param input input line of the command
  * \param len of the command
+ * \param thread context
  * \param cmd_descriptor when the command has been parsed
  *
  * \retval 1 when the command is parsed, 0 otherwise
  */
-static int FTPParseRequestCommand(uint8_t *input, uint32_t input_len, const FtpCommand **cmd_descriptor)
+static int FTPParseRequestCommand(uint8_t *input, uint32_t input_len,
```

Comment:
minor: we generally order the args global data -> thread data -> local data. That would mean here ```td``` would be the first arg.